### DISCUSSION TITLE
Freac 1.1.6. freezes

### DISCUSSION THREAD

**Discussion by BaudolinO22** [enzo1982/freac#300] [Help & Support]
Dear friends
I have version 1.1.6 but today when I try to add new files the window freezes and I have to close freac.
What I have to do?
I've tried to uninstall and reinstall and to downgrade to 1.1.5 but it does not work.
I have Win 10 home updated
Thanks, Paolo

**Comment by BaudolinO22** [comment]
![Appunti01](https://user-images.githubusercontent.com/104224395/164803116-b05a3f11-8707-40a1-a9eb-118595d3f729.jpg)
Thats what happens...

**Reply by enzo1982** [reply]
That's the standard Windows _Open file_ dialog freezing there. That's most likely not caused by fre:ac, but by some system extension hooking into Windows Explorer.

There are hundreds of programs and tools installing such Explorer extensions. Impossible for me to say what is causing this on your system.

What you can try is removing the fre:ac config file. It should be located at: `C:\Users\<your user name>\AppData\Roaming\freac\freac.xml` That file stores the last folder from which files were added so fre:ac can instruct the _Open file_ dialog to open that folder again. Maybe that last location is causing problems with some Explorer extension, e.g. if it is on a drive that is not currently connected or a network resource that is not currently reachable.

**Comment by BaudolinO22** [comment]
and I only have to close Freac...
